### PR TITLE
feat(ci): replaced GHCR_PR_TOKEN with GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by aali309**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits): `git commit -S -m "YOUR_COMMIT_MESSAGE"`
_______________________________________________

Fixes: #613 

## Description of the change:
Replace GHCR_PR_TOKEN with GITHUB_TOKEN

## Motivation for the change:
Raised initially by Thuan

## How to manually test:
1. *Insert steps here...*
2. *...*

### CODE REVIEW

**Review by andrewazores — COMMENTED** [review]
Any sample run to verify that no additional permissions need to be specified/granted to this token?

**Comment by tthvo** [comment]
We will also need to grant `cryostat-operator` repo access (i.e. write) to the `-operator`, `-operator-scorecard` and `-operator-bundle` "packages" on `ghcr.io`.

**Comment by aali309** [comment]
https://github.com/aali309/cryostat-operator/actions/runs/6351069848

**Comment by andrewazores** [comment]
> We will also need to grant `cryostat-operator` repo access (i.e. write) to the `-operator`, `-operator-scorecard` and `-operator-bundle` "packages" on `ghcr.io`.

Nice, I think I have set that up properly now.

File: .github/workflows/dependent-issues.yml [hunk 1]
```diff
@@ -12,14 +12,18 @@ on:
       - edited
       - reopened
       - synchronize
-
+
```
Comment (tthvo) [inline .github/workflows/dependent-issues.yml:15]: Remove space here?

**Review by ebaron — APPROVED** [review]
Looks good to me, thanks @aali309!

**Pull request merged**